src/components/LoginSite.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+import { firebase, auth } from "../firebase";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { Link } from "react-router-dom";
+
+export class LoginSite extends Component {
+  state = {
+    email: "",
+    password: ""
+  };
+
+  handleChange = e => {
+    this.setState({
+      [e.target.name]: e.target.value
+    });
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+    const { email, password } = this.state;
+    firebase
+      .auth()
+      .signInWithEmailAndPassword(email, password)
+      .then(user => {
+        this.props.history.push("/");
+      })
+      .catch(error => {
+        this.setState({ error });
+      });
+    console.log("zalogowano");
```

Comment:
This is true in between lines 24 and 26 - not here :)

Reply:
But it works in this placem I checked :D I truly don't know, how to put it between 24 - 26.